src/OpenTelemetry.AutoInstrumentation/CallTarget/CallTargetInvoker.cs

```diff
@@ -59,7 +59,7 @@ public static class CallTargetInvoker
     {
```

Comment:
What is the reason to keep old (without ref) version of the methods?

Reply:
In the upstream scenario, this was required because the user could download the SDK and reference the older version, so the old one is needed for backward compatibility. I believe that is not needed in this repo so I can do this in this PR

Reply:
We can merge it as it is and cleanup in follow up PR. Whatever is easier for you.

Reply:
Let's do separate cleanup, I think that will be easier for history tracking

Reply:
Can you create separate GH issue? I suppose it is related to https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2280#discussion_r1156730613

Reply:
#2424